app/assets/javascripts/spree/backend/solidus_reviews.js

```diff
@@ -1,7 +1,9 @@
-//= require jquery.rating
+//= require jquery-bar-rating
 //= require spree/backend
 
-// Navigating to a page with ratings via TurboLinks shows the radio buttons
-$(document).on('page:load', function () {
-  $('input[type=radio].star').rating();
+$(document).on('ready', function () {
+  $.each($("select.review-stars"), function(index, value) {
```

Comment:
To be honest, me too. This select is located in the **'spree/reviews/stars'** partial. I've just updated old JS files with new code